ui/pages/settings/settings.container.js

```diff
@@ -60,7 +61,7 @@ const mapStateToProps = (state, ownProps) => {
   const {
     metamask: { currencyRates },
   } = state;
-
+  const remoteFeatureFlags = getRemoteFeatureFlagsByName(state);
```

Comment:
should we pass a feature flag name in here?

Reply:
reverted this selector and name is not passed here any more 89fe7aced2e9557d1841598bead82c5ac897cfb9